### PR TITLE
Fix false naming escaping in JavaScript 

### LINKED ISSUE #1940: JS. Function call has extra apostrophes
**Description**

Function call `na.'double'()` from a generated test has extra apostrophes. 

**To Reproduce**

1. [Install the latest IU plugin built from unit-test-bot/rc3102023 branch in IntelliJ Idea Ultimate 2022.2.4](https://github.com/UnitTestBot/UTBotJava/actions/runs/4405655378) - take **`utbot-intellij-IU-2023.3.*`** artefact 
2. Create a JavaScript project with sample code
3. Copy samples folder from `utbot-js` module and paste it into this project
4. Generate tests for Na.double() function in `samples/scenarioMultyClassNoTopLevel.js`

**Expected behavior**

Function calls in the generated tests are supposed to be correctly rendered and compillable.
Expected - compiling, test passes with this code:

```javascript
na.double()
```

**Actual behavior**

Function call `na.'double'()` from a generated test has extra apostrophes. 
Actual piece of code:

```javascript
na.`double`()
```

**Visual proofs (screenshots, logs, images)**

The following test is generated:

~~~javascript
const fileUnderTest = require("./../samples/scenarioMultyClassNoTopLevel.js")
const assert = require("assert")

///region Test suites for executable Na.double

///region

it("testDouble1", function ()
 {
    let na = new fileUnderTest.Na("\n\t\n")
    
    let actual = na.`double`()
    
    assert.deepEqual(0, actual)
}
)
///endregion

///endregion
~~~

**Environment**

Windows 10

**Additional context**

The function expects a number, but it is given a String.

### PR DESCRIPTION
**Pull request by rudolf101**
## Description

Fixes #1940 

### Manual tests

Various scenarios have been tested. You can find the examples in the folder `/utbot-js/samples' and try to run UTBot.

## Self-check list

Check off the item if the statement is true. Hint: [x] is a marked item.

Please do not delete the list or its items.

- [x] I've set the proper **labels** for my PR (at least, for category and component).
- [x] PR **title** and **description** are clear and intelligible.
- [ ] I've added enough **comments** to my code, particularly in hard-to-understand areas.
- [ ] The functionality I've repaired, changed or added is covered with **automated tests**.
- [ ] **Manual tests** have been provided optionally.
- [ ] The **documentation** for the functionality I've been working on is up-to-date.

### CODE REVIEW

File: utbot-js/src/main/kotlin/framework/codegen/model/constructor/visitor/CgJsRenderer.kt [hunk 1]
```diff
@@ -409,8 +408,7 @@ internal class CgJsRenderer(context: CgRendererContext, printer: CgPrinter = CgP
         print("${exception.name.escapeNamePossibleKeyword()}: ${exception.type}")
     }
 
-    override fun escapeNamePossibleKeywordImpl(s: String): String =
-        if (isLanguageKeyword(s, context.cgLanguageAssistant)) "`$s`" else s
+    override fun escapeNamePossibleKeywordImpl(s: String): String = ""
```
Comment (Markoutte) [inline utbot-js/src/main/kotlin/framework/codegen/model/constructor/visitor/CgJsRenderer.kt:411]: Why is empty string return?

**Review by Markoutte — CHANGES_REQUESTED** [review]

File: utbot-js/src/main/kotlin/framework/codegen/model/constructor/visitor/CgJsRenderer.kt (hunk 1 above)
Comment (rudolf101) [reply utbot-js/src/main/kotlin/framework/codegen/model/constructor/visitor/CgJsRenderer.kt:411]: This is a mockup for API, JavaScript does not support keyword escaping for function calls

Comment (Markoutte) [reply utbot-js/src/main/kotlin/framework/codegen/model/constructor/visitor/CgJsRenderer.kt:411]: I ran with this patch the code that has the problem and got this:

```js
it("testDouble1", function ()
 {
    let  = new fileUnderTest.Na("\n\t\n")
    
    let  = .()
    
    (0, )
}
)
```

**Review by Markoutte — CHANGES_REQUESTED** [review]
I reopened conversation. Please, see the problem I have

File: utbot-js/src/main/kotlin/framework/codegen/model/constructor/visitor/CgJsRenderer.kt (hunk 1 above)
Comment (zishkaz) [reply utbot-js/src/main/kotlin/framework/codegen/model/constructor/visitor/CgJsRenderer.kt:411]: Oops, my bad, will be properly fixed soon

**Review by Markoutte — APPROVED** [review]

**Pull request merged**